massa-consensus-worker/src/state/graph.rs

```diff
@@ -43,6 +63,9 @@ impl ConsensusState {
             if let Some(BlockStatus::Active { a_block: ab, .. }) =
                 self.blocks_state.get_mut(&ancestor_h)
             {
+                if ab.is_final {
```

Comment:
As it seems this was previously missed, maybe add a comment here

Reply:
done